### PR TITLE
Add 'opentelekomcloud_ims_image_v2' resource and docs

### PR DESCRIPTION
**Pull request by ShiChangkuo**
The changes are as follows:
1. pull github.com/huaweicloud/golangsdk;
2. define the 'opentelekomcloud_ims_image_v2' resource;
3. add test case for TestACC;
4. add docs.

### CODE REVIEW

File: opentelekomcloud/resource_opentelekomcloud_images_ims_v2.go [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+		tags = append(tags, tagRequest)
+	}
+	return tags
+}
+
+func resourceImagesImsV2Create(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
+	ims_Client, err := config.imageV2Client(GetRegion(d, config))
+	if err != nil {
+		return fmt.Errorf("Error creating OpenTelekomCloud EVS storage client: %s", err)
```
Comment (niuzhenguo) [inline opentelekomcloud/resource_opentelekomcloud_images_ims_v2.go]: should be IMS client here.

File: opentelekomcloud/resource_opentelekomcloud_images_ims_v2.go [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+	}
+
+	img, err := getCloudimage(ims_Client, d.Id())
+	if err != nil {
+		return fmt.Errorf("Image %s not found: %s", d.Id(), err)
+	}
+	log.Printf("[DEBUG] Retrieved Image %s: %#v", d.Id(), img)
+
+	d.Set("name", img.Name)
+	d.Set("id", img.ID)
```
Comment (niuzhenguo) [inline opentelekomcloud/resource_opentelekomcloud_images_ims_v2.go]: id should be set by d.SetId(img.ID)

File: website/docs/r/images_ims_v2.html.markdown [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+sidebar_current: "docs-opentelekomcloud-resource-images-ims-v2"
+description: |-
+  Manages a V2 Image resource within OpenTelekomCloud IMS.
+---
+
+# opentelekomcloud\_images\_ims_v2
+
+Manages a V2 Image resource within OpenTelekomCloud IMS.
+
+## Example Usage
```
Comment (niuzhenguo) [inline website/docs/r/images_ims_v2.html.markdown]: better to also give an example with OBS here.

File: opentelekomcloud/provider.go [hunk 4]
```diff
@@ -250,6 +250,7 @@ func Provider() terraform.ResourceProvider {
 			"opentelekomcloud_identity_group_v3":                  resourceIdentityGroupV3(),
 			"opentelekomcloud_identity_group_membership_v3":       resourceIdentityGroupMembershipV3(),
 			"opentelekomcloud_images_image_v2":                    resourceImagesImageV2(),
+			"opentelekomcloud_images_ims_v2":                      resourceImagesImsV2(),
```
Comment (niuzhenguo) [inline opentelekomcloud/provider.go]: how about naming it as opentelekomcloud_ims_image_v2, just like ecs_instance_v1, evs_volume_v3 , SERVICE_RESOURCE pattern.

**Review by niuzhenguo — COMMENTED** [review]
Thanks for working on this, please see comments inline.

File: opentelekomcloud/provider.go (hunk 4 above)
Comment (ShiChangkuo) [reply opentelekomcloud/provider.go]: yep, it looks better and I will rename the resource to opentelekomcloud_ims_image_v2

File: opentelekomcloud/resource_opentelekomcloud_images_ims_v2.go (hunk 1 above)
Comment (ShiChangkuo) [reply opentelekomcloud/resource_opentelekomcloud_images_ims_v2.go]: @niuzhenguo Thanks for your comments, please review it in the new commits.

File: opentelekomcloud/resource_opentelekomcloud_images_ims_v2.go (hunk 2 above)
Comment (ShiChangkuo) [reply opentelekomcloud/resource_opentelekomcloud_images_ims_v2.go]: ok, it has been fixed.

File: website/docs/r/images_ims_v2.html.markdown (hunk 3 above)
Comment (ShiChangkuo) [reply website/docs/r/images_ims_v2.html.markdown]: @niuzhenguo Thanks for your comments, please review it in the new commits.

**Pull request merged**